Imageview/app/src/main/java/com/mycompany/imageview/Library_fragment.java

```diff
@@ -0,0 +1,143 @@
+        View rootview = inflater.inflate(R.layout.fragment_library_fragment, container, false);
+
+        db = new MySQLiteHelper(getActivity());
+        Image_list = db.get_all_images();
+
+        create_webview(rootview);
+        create_button(rootview, "google");
+        create_button(rootview, "next");
+        create_button(rootview, "previous");
+        create_button(rootview, "delete");
+
+        return rootview;
+    }
+
+
+    public  void create_webview(View view){
+        mWebView = (WebView)view.findViewById(R.id.library_webview);
+
+        // Enable Javascript
+        WebSettings webSettings = mWebView.getSettings();
+        webSettings.setJavaScriptEnabled(true);
+        webSettings.setBuiltInZoomControls(true);
+        webSettings.setUseWideViewPort(true);
+        webSettings.setLoadWithOverviewMode(true);
+        link ="http://www.canon-europe.com/images/Android-logo_tcm13-1232684.png";
+        mWebView.loadUrl(link);
+    }
+
+
+    public void create_button(View v, String button){
+        Button button_search;
+        button_search=(Button)v.findViewById(R.id.button_search);
+        button_search.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                transitionToFragment();
+            }
+        });
+
+        if (button.equals("next") ){
+            Button button_next;
+            button_search = (Button) v.findViewById(R.id.button_library_next);
+
+            button_search.setOnClickListener(new View.OnClickListener() {
+                @Override
+                public void onClick(View v) {
+                    Log.d(TAG, "next ");
+                    pointer+=1;
+                    reload_webview();
+                }
+            });
+        }
+        if (button.equals("previous") ){
+            Button button_previous;
+            button_search = (Button) v.findViewById(R.id.button_library_previous);
+
+            button_search.setOnClickListener(new View.OnClickListener() {
+                @Override
+                public void onClick(View v) {
+                    Log.d(TAG, "previous");
+                    pointer-=1;
+                    reload_webview();
+                }
+            });
+        }
+        if (button.equals("delete") ){
+            Button button_delete;
+            button_search = (Button) v.findViewById(R.id.button_delete);
+
+            button_search.setOnClickListener(new View.OnClickListener() {
+                @Override
+                public void onClick(View v) {
+                    try {
+                        db.deleteImage(Image_list.get(pointer % Image_list.size()));
+                    }
+                    catch (Exception e){
+                        Log.d(TAG, e.getMessage());
+                    };
+                    Image_list = db.get_all_images();
+                }
+            });
+        }
+
+    }
```

Comment:
Interesting way of creating a single method for all button creation. It might be cleaner if each was its own method.